spec/factories/academies_api/establishment.rb

```diff
@@ -9,6 +9,7 @@
     phase { "Secondary" }
     diocese_name { "Diocese of West Placefield" }
     region_name { "West Midlands" }
+    region_code { "F" }
```

Comment:
I wish we could also use the friendly names here as well? I wonder if there is a way to map them?

Reply:
This is a mock up of how the codes are output by the API, so I don't think it's appropriate to use the friendly name here?